hexa/metadata/models.py

```diff
@@ -0,0 +1,32 @@
+import uuid
+
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+
+from hexa.core.models.base import Base
+
+
+class MetadataObject(Base):
+    content_type = models.ForeignKey(
+        ContentType, null=True, blank=True, on_delete=models.CASCADE
+    )
+    object_id = models.UUIDField(default=uuid.uuid4, editable=False)
+    data_object = GenericForeignKey("content_type", "object_id")
+
+
+class MetadataAttribute(Base):
+    key = models.CharField(max_length=255, unique=True)
```

Comment:
It means that if you have multiple files that have a column named "n_people" the profiling keys will throw an error since they won't be unique. The _uniqueness_ of the key is for (MetadataObject, key)